pest-ion/src/bin/pest2ion/main.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates.
+
+use anyhow::Result;
+use clap::{crate_authors, crate_description, crate_version, App, Arg, ArgGroup};
+use ion_rs::value::writer::{ElementWriter, Format, TextKind};
+use pest_ion::{from_read, TryPestToElement};
+use std::fs::File;
+use std::io::{stdin, stdout, Write};
+use std::path::Path;
+
+fn main() -> Result<()> {
+    let matches = App::new("Pest to Ion Converter")
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(crate_description!())
+        .arg(
+            Arg::with_name("FILE")
+                .help("The input file to parse (defaults to STDIN)")
+                .index(1),
+        )
+        .arg(
+            Arg::with_name("OUTPUT_FILE")
```

Comment:
I think `FILE` above is all caps because it's an anonymous argument. If that's true, then this should be `output-file`?

Reply:
It actually appears in the help, and this is somewhat consistent to help notation I've seen elsewhere:

```
❯ cargo run --package pest-ion --bin pest2ion -- --help
   Compiling pest-ion v0.0.0 (/home/ANT.AMAZON.COM/almann/CLionProjects/partiql-lang-rust/pest-ion)
    Finished dev [unoptimized + debuginfo] target(s) in 0.88s
     Running `target/debug/pest2ion --help`
Pest to Ion Converter 0.0.0
PartiQL Team <partiql-team@amazon.com>
A simple Pest grammar to Ion converter

USAGE:
    pest2ion [FLAGS] [OPTIONS] [FILE]

FLAGS:
    -b, --binary     Generate Ion binary
    -h, --help       Prints help information
    -p, --pretty     Generate Ion pretty printed text
    -t, --text       Generate Ion text (default)
    -V, --version    Prints version information

OPTIONS:
    -o, --output <OUTPUT_FILE>    Writes output to the given file (defaults to STDOUT)

ARGS:
    <FILE>    The input file to parse (defaults to STDIN)
```

Reply:
You're right, now that I see it in the context of the output it does look right.